src/commands/createContainerApp/IngressVisibilityStep.ts

```diff
@@ -17,6 +17,6 @@ export class IngressVisibilityStep extends AzureWizardPromptStep<IContainerAppCo
     }
 
     public shouldPrompt(context: IContainerAppContext): boolean {
-        return context.enableIngress === true;
+        return context.enableExternal === undefined;
```

Comment:
Another nit, but just to minimize the potential for side effects, I would probably keep the original statement as well.  Maybe something like:

`return context.enableIngress === true && context.enableExternal === undefined;`